index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+      }
+
+      body {
+        background: #0b2429;
+
+        min-width: 100vw;
+        min-height: 100vh;
+
+        display: grid;
+        place-items: center;
+
+        overflow: hidden;
+      }
+
+      .circle-two {
+        background: #998235;
+        width: 120px;
+        height: 120px;
+
+        border-radius: 100px 0px 100px 100px;
+
+        position: relative;
+
+        transform: scale(0.85);
+      }
+
+      .circle-two::before,
+      .circle-two::after {
+        content: "";
+
+        display: inline-block;
+
+        background: #f3ac3c;
+        width: 120px;
+        height: 120px;
+      }
+
+      .circle-two::after {
+        border-radius: 100px 0px 100px 100px;
+
+        position: absolute;
+        left: 0;
+        transform: translate(-50%, 50%);
+      }
+
+      .circle-two::before {
+        border-radius: 100px;
+
+        transform: translate(50%, -50%);
+
+        /* z-index: -5; */
```

Comment:
it's always better to remove commented code to make reviewing process faster.

Reply:
ok will do that